plugins/filter_kubernetes_sumologic.rb

```diff
@@ -147,7 +144,6 @@ def filter(tag, time, record)
         else
           sumo_metadata[:category] = (annotations['sumologic.com/sourceCategory'] % k8s_metadata).prepend(@source_category_prefix)
         end
-        sumo_metadata[:category].gsub!('-', @source_category_replace_dash)
 
```

Comment:
This should not be removed as this is a heavily utilized and valuable feature today.  Did you really want to remove this?